pkg/identityManager/identityManager_test.go

```diff
@@ -102,6 +122,64 @@ var _ = Describe("IdentityManager", func() {
 		}
 		// Make sure the namespace has been cached for subsequent retrieval.
 		Eventually(func() (*v1.Namespace, error) { return namespaceManager.GetNamespace(remoteCluster) }).Should(Equal(namespace))
+
+		// Certificate Secret Section
+		apiServerConfig = apiserver.Config{Address: "127.0.0.1", TrustedCA: false}
+		Expect(apiServerConfig.Complete(restConfig, client)).To(Succeed())
+
+		signingIdentityResponse = responsetypes.SigningRequestResponse{
+			ResponseType: responsetypes.SigningRequestResponseCertificate,
+			Certificate:  []byte("cert"),
+		}
+
+		secretIdentityResponse, err = auth.NewCertificateIdentityResponse(
+			"remoteNamespace", &signingIdentityResponse, apiServerConfig)
+		Expect(err).To(BeNil())
```

Comment:
```suggestion
		Expect(err).ToNot(HaveOccurred())
```
the same for the others